CONTRIBUTING.md

```diff
@@ -6,36 +6,72 @@ For major changes, please open an issue to discuss what you would like to change
 As of writing this document, the project is still in a pre-1.0 state.
 This means that the API is not yet stable and breaking changes may occur frequently.
 
-## Pull Requests
+## Prerequisites
+
+- [`rust`](https://www.rust-lang.org/tools/install)
+- [`docker`](https://docs.docker.com/engine/install/)
+  - You will need this to test the CI/CD pipelines locally or to use Earthly to run various project commands.
+  - You may need to start the docker daemon before running any commands. You can do this with `sudo systemctl start docker`.
+  - You may need to add your user to the `docker` group to run docker commands without `sudo`. You can do this with `sudo usermod -aG docker $USER`.
+- [`hermit`](https://cashapp.github.io/hermit/usage/get-started/)
+  - This tool provides binaries you may want to have at hand to work on this repo.
+  - You can see the full list of tools in the `./bin` directory in the repo root.
+  - If you do not want to use hermit, you can install each tool manually.
+  - If you wish to use some tools from hermit, you may execute them directly with `./bin/<tool>`.
+  - (Recommended) If you wish to use all tools provided by earthly, we recommend installing the shell hooks (see [here](https://docs.earthly.dev/guides/shell-hooks) for more information).
```

Comment:
I think the link for shell hooks here is wrong; I think you want https://cashapp.github.io/hermit/usage/shell/ instead 

Also, since I'm not really good with these things, it is nice to know why it's necessary for me to be doing them so maybe add a line like "Doing this will prevent you from needing to reactivate your hermit environment every time you re-open clam" (or something like that--not sure if my suggestion is totally accurate)